libcudacxx/include/cuda/std/__mdspan/aligned_accessor.h

```diff
@@ -0,0 +1,101 @@
+//@HEADER
+// ************************************************************************
+//
+//                        Kokkos v. 4.0
+//       Copyright (2022) National Technology & Engineering
+//               Solutions of Sandia, LLC (NTESS).
+//
+// Under the terms of Contract DE-NA0003525 with NTESS,
+// the U.S. Government retains certain rights in this software.
+//
+// Part of Kokkos, under the Apache License v2.0 with LLVM Exceptions.
+// See https://kokkos.org/LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+// SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES.
+//
+// ************************************************************************
+//@HEADER
+
+#ifndef _LIBCUDACXX___ALIGNED_ACCESSOR_H
```

Comment:
Other headers also include the term:
```suggestion
#ifndef _LIBCUDACXX___MDSPAN_ALIGNED_ACCESSOR_H
```